src/OpenTelemetry.Api.ProviderBuilderExtensions/OpenTelemetry.Api.ProviderBuilderExtensions.csproj

```diff
@@ -8,10 +8,11 @@
     <AnalysisLevel>latest-all</AnalysisLevel>
   </PropertyGroup>
 
-  <!--Do not run ApiCompat as this package has never released a stable version.
+  <!--Do not run ApiCompat nor Package Baseline Validation as this package has never released a stable version.
   Remove this property once we have released a stable version.-->
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <RunApiCompat>false</RunApiCompat>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
Unfortunately this won't work as you are setting the property in the inner build (`net6.0`) while Package Validation runs as part of the `Pack` target which runs in the outer build. This might sound complicated but it essentially means that Package Validation looks at the entire package which requires all inner builds to be completed. An inner build is a build for one TFM for multi-targeting projects.

That `DisablePackageBaselineValidation` switch can't be set in an inner build. Move it out of that property group.

Reply:
Got it, thanks! Moved it to a separate `PropertyGroup`.